lib/widgets/home.dart

```diff
@@ -259,6 +260,7 @@ void _showMainMenu(BuildContext context, {
     // TODO(#1094): Users
     const _MyProfileButton(),
     const _SwitchAccountButton(),
+    const _AboutZulipButton(),
```

Comment:
nit: Let's move this down a few lines, so it comes after the TODOs for buttons that should appear earlier in the list. Probably just above `// TODO(#1095): VersionInfo` is the right place.

```
nav: Add about-zulip button to menu bar
```

Also, a commit-message nit: this isn't really a "menu bar"; you can just call it "the main menu".

Reply:
Another commit-message nit:

```
nav: Add about-zulip button to menu bar

This is an implementation for issue #1128

This commit includes localizations and tests for the button as well.
```

Just say `Fixes: #1128`; you don't need a whole sentence for that.

Also no need to mention localization and tests. Those are always required, so they don't need a special mention in the commit message.